packages/react-storage/src/components/FileUploader/utils/getInput.ts

```diff
@@ -8,6 +7,8 @@ import { ProcessFile } from '../types';
 import { resolveFile } from './resolveFile';
 import { PathCallback, PathInput } from './uploadFile';
 
+type StorageAccessLevel = 'guest' | 'protected' | 'private';
```

Comment:
Nit: would it be better to declare StorageAccessLevel here or import it from `FileUploader/types`?

Reply:
Yeah I think it would make sense to import it from the types file, updated to do that.